demisto_sdk/commands/prepare_content/preparers/incident_to_alert.py

```diff
@@ -0,0 +1,260 @@
+import re
+import copy
+import logging
+from typing import Any, List, Tuple
+
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.objects.content_item import ContentItem
+
+logger = logging.getLogger("demisto-sdk")
+
+NOT_WRAPPED_RE_MAPPING = {
+    rf"(?<!<-){key}(?!->)": value
+    for key, value in {
+        "incident": "alert",
+        "Incident": "Alert",
+        "incidents": "alerts",
+        "Incidents": "Alerts",
+        "INCIDENT": "ALERT",
+        "INCIDENTS": "ALERTS",
+    }.items()
```

Comment:
```suggestion
    for key, value in TABLE_INCIDENT_TO_ALERT.items()
```
let's import it from `tools.py` - single source of truth. 
Actually, it should probably be saved either here or under `constants.py`